packages/create-cloudflare/turbo.json

```diff
@@ -26,7 +26,8 @@
 				"E2E_EXPERIMENTAL",
 				"TEST_PM"
 			],
-			"dependsOn": ["build"]
+			"dependsOn": ["build"],
+			"outputs": [".e2e-logs", ".e2e-logs-experimental"]
```

Comment:
This is the key to the logs being available even if turbo decides to use the cache hit.